packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -7,42 +7,65 @@
+ * @prop --calcite-tile-selection-icon-color-hover: Specifies the color of the selection icon in the component on hover.
+ * @prop --calcite-tile-shadow: Specifies the box-shadow of the component.
  */
 
+/** Internal variables
+  * --calcite-internal-tile-border-color
+  * --calcite-internal-tile-description-text-color
+  * --calcite-internal-tile-heading-text-color
+  * --calcite-internal-tile-icon-color
+  * --calcite-internal-tile-outline-color
+  * --calcite-internal-tile-selection-icon-color
+  * --calcite-internal-tile-shadow
+  */
+
 :host {
-  --calcite-tile-background-color: var(--calcite-color-foreground-1);
-  --calcite-tile-border-color: var(--calcite-color-border-2);
-  --calcite-tile-description-text-color: var(--calcite-color-text-3);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-2);
-  --calcite-icon-color: var(--calcite-color-text-3);
+  --calcite-internal-tile-border-color: var(--calcite-tile-border-color, var(--calcite-color-border-2));
+  --calcite-internal-tile-description-text-color: var(
+    --calcite-tile-description-text-color,
+    var(--calcite-color-text-3)
+  );
+  --calcite-internal-tile-heading-text-color: var(--calcite-tile-heading-text-color, var(--calcite-color-text-2));
+  --calcite-internal-tile-icon-color: var(--calcite-color-text-3);
 
   box-sizing: border-box;
   display: inline-block;
 }
 
 .container {
-  background-color: var(--calcite-tile-background-color);
+  --calcite-internal-tile-outline-color: var(--calcite-tile-border-color, var(--calcite-internal-tile-border-color));
+  --calcite-internal-tile-select-icon-color: var(--calcite-tile-selection-icon-color, var(--calcite-color-text-3));
+
+  background-color: var(--calcite-tile-background-color, var(--calcite-color-foreground-1));
   block-size: var(--calcite-container-size-content-fluid);
   box-sizing: border-box;
   inline-size: var(--calcite-container-size-content-fluid);
-  outline: var(--calcite-border-width-sm, 1px) solid var(--calcite-tile-border-color);
+  outline: var(--calcite-border-width-sm, 1px) solid var(--calcite-internal-tile-outline-color);
   user-select: none;
+  box-shadow: var(--calcite-tile-shadow, var(--calcite-internal-tile-shadow, none));
+
   &.interactive {
     cursor: pointer;
+
     &:hover,
     &:focus,
     &.selected {
-      outline-color: var(--calcite-color-brand);
+      --calcite-internal-tile-outline-color: var(--calcite-tile-border-color, var(--calcite-color-brand));
+      --calcite-internal-tile-select-icon-color: var(
+        --calcite-tile-selection-icon-color-hover,
+        var(--calcite-color-brand)
+      );
       position: relative;
-      .selection-icon {
-        --calcite-icon-color: var(--calcite-color-brand);
-      }
     }
     &.selected {
       z-index: var(--calcite-z-index);
     }
     &:focus {
-      box-shadow: inset 0 0 0 1px var(--calcite-color-brand);
+      --calcite-internal-tile-shadow: inset 0 0 0 1px var(--calcite-color-brand);
```

Comment:
I don't think we should be allowing changes to the full shadow. Instead we should only allow changes to `--calcite-tile-shadow-color`

Reply:
good catch.  I wasn't sure about this shadow token.  I'll rename it to shadow-color and update its usage.

Reply:
Is this really a shadow in practice to a user? I'd expect this to use the `--calcite-tile-border-color`. 

And then a separate, `--calcite-tile-shadow` that defaults to the "none shadow" to allow for customization.

Reply:
@macandcheese is correct. `box-shadow` is used as a border here and not the component's shadow. Other props are used for borders as well (I think `themed` [covers all](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/tests/commonTests/themed.ts#L402)).

Reply:
Thanks for the catch @macandcheese! Yes agreed this should be using `--calcite-tile-border-color`

Reply:
Realizing that the best way to set a shadow is to just let the user do `calcite-tile { box-shadow: "whatever" }` because the `.container` div needs to control its own `box-shadow` property for the `inset` focus border, so creating a token would be superfluous.

Reply:
But is that how we want people to adjust these as far as consistency? 

It feels a bit weird to be able to do most of the theming via css property but have to set one rule a different way. Just IMO though 😅 

You could pass it in as a second box shadow argument perhaps if you didn’t want to restructure?

Reply:
Created a Teams thread to continue this convo: https://teams.microsoft.com/l/message/19:8be7cb0052e045b2a3138b84781a321a@thread.skype/1728324717995?tenantId=aee6e3c9-711e-4c7c-bd27-04f2307db20d&groupId=56fae21a-9407-4943-859f-a9bfcf0bbad3&parentMessageId=1728324717995&teamName=Calcite%20Design%20System&channelName=Core%20-%20Design%20Tokens&createdTime=1728324717995&ngc=true

Reply:
Opted for creating a `--calcite-tile-shadow` token that defaults to `none` using the `--calcite-shadow-none` reference just like Card.